### ISSUE TITLE
Can't get Windows app working

### ISSUE THREAD

**Issue by hugegit** [apprenticeharper/DeDRM_tools#808]
Warning: the answer to my question is likely braindead-simple - I have a background in programming but this is the first time I've touched Python. If there's a different place I should be posting this, please advise...

I installed Python 2.7 on Windows 10 64bit - no apparent problems with install
I installed PyCrypto:  pycrypto-2.6.1.win-amd64-py2.7.msi
I downloaded "Alfs DeDRM tools for Calibre", found the DeDRM_Windows_Application folder and copied the DeDRM_App folder elsewhere.
I made a shortcut pointing to DeDRM_Drop_Target.bat

When I drag .mobi or .azw files and drop them onto the shortcut, I see a very brief command window open but it disappears before I can see the few lines of text.  

I do not see any new .mobi/.azw file created anywhere, or any evidence that anything has happened.

Again, please pardon my Python ignorance, but can you tell me what might be going wrong or what to try next?

**Comment by Engineer-of-Stuff** [comment]
Try going to the `DeDRM_lib` folder and running `DeDRM_App.pyw` in python2.

**Comment by ElleKayEm** [comment]
Personally I find it a lot easier to just use calibre and the plugin.  But if you prefer the standalone app, some people have had luck with installing pylzma.  Or you could try going back to version 6.5.5 which doesn't require that.  Also look at this section in the FAQs:

https://github.com/apprenticeharper/DeDRM_tools/blob/master/FAQs.md#the-windows-application

**Comment by hugegit** [comment]
Thanks for both replies! Downgrading to 6.5.5 seems to have fixed the problem. 

In case it's of interest, I'll report the error reported when I ran DeDRM_App.pyw (from 6.6.3) in a windows command shell (which I am guessing indicates a problem with PyCrypto):

...\dedrm_lib> python DeDRM_App.pyw
Using Library AlfCrypto DLL/DYLIB/SO
Traceback (most recent call last):
  File "DeDRM_App.pyw", line 69, in <module>
    from scriptinterface import decryptepub, decryptpdb, decryptpdf, decryptk4mobi
  File "E:\Downloads from C drive\Alfs DeDRM tools for Calibre\DeDRM_Windows_Application\DeDRM_App\dedrm_lib\lib\scriptinterface.py", line 14, in <module>
    import k4mobidedrm
  File "E:\Downloads from C drive\Alfs DeDRM tools for Calibre\DeDRM_Windows_Application\DeDRM_App\dedrm_lib\lib\k4mobidedrm.py", line 94, in <module>
    import kfxdedrm
  File "E:\Downloads from C drive\Alfs DeDRM tools for Calibre\DeDRM_Windows_Application\DeDRM_App\dedrm_lib\lib\kfxdedrm.py", line 20, in <module>
    import ion
  File "E:\Downloads from C drive\Alfs DeDRM tools for Calibre\DeDRM_Windows_Application\DeDRM_App\dedrm_lib\lib\ion.py", line 20, in <module>
    from Crypto.Cipher import AES
  File "C:\Users\Laurence\AppData\Roaming\Python\Python27\site-packages\Crypto\Cipher\AES.py", line 50, in <module>
    from Crypto.Cipher import _AES
ImportError: DLL load failed: %1 is not a valid Win32 application.

**Comment by fireattack** [comment]
I encountered the same issue today.

Installing x86 (32-bit) Python 2 fixed it.

**Comment by cannothing** [comment]
I am not sure, whether I have the same issue or it is a different.
I am using DeDRM_Drop_Target for two years without any problem. 
This morning only the cmd-window( PyCrypto 2.6 (or later) for Windows and Python 2.7) appeared and disappeared. So I tried a file, from which I removed the DRM yesterday successfully. Same issue.
So I uninstalled and reinstalled Phyton 2.7 and  PyCrypto 2.6 (or later) for Windows and Python 2.7. 
Now I can remove DRM again.
But perhaps there is a less time-consuming method than uninstalling and reinstalling.

**Comment by cannothing** [comment]
I am not sure, whether I have the same issue or it is a different.
I am using DeDRM_Drop_Target for two years without any problem. 
This morning only the cmd-window for Windows and Python 2.7) appeared and disappeared. So I tried a file, from which I removed the DRM yesterday successfully. Same issue.
So I uninstalled and reinstalled Phyton 2.7 and  PyCrypto 2.6 (or later) for Windows and Python 2.7. 
Now I can remove DRM again.
But perhaps there is a less time-consuming method than uninstalling and reinstalling.
